covid_graphs/README.md

```diff
@@ -0,0 +1,31 @@
+# Graphs and visualizations related to COVID-19
+
+## Running the whole server
+
+In the root directory of the repository, run `docker-compose`.
+```sh
+docker-compose up --build # Optionally add -d for deamon
+```
+You can then access the server locally at `http://127.0.0.1:8081`.
+
+
+### Running without Docker
+Alternatively, you can run the server locally without Docker. Create a virtual
+environment, install the `covid_graphs` package an run the server.
+```sh
+python3.7 -m venv your/path/to/venv
+source your/path/to/venv/bin/activate
+pip install -e .
+server.py data build/results-poly.pb build/results-exp.pb
+```
```

Comment:
Expanded on this section.

Reply:
This part will use the port 8080, which is confusing, because it's different from `docker-compose`. I suggest changing the port inside server.py to 8081 as well (you'll need to modify docker-compose as well).